.travis.yml

```diff
@@ -72,7 +72,7 @@ jobs:
         # generated contract code which caused the whole file to be printed as part of the warning
         # message which made the build fail (probably travis related).
         - env CARGO_INCREMENTAL=0 RUSTFLAGS="-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off -Awarnings" cargo +nightly test
-        - ./grcov --branch --ignore-not-existing --llvm --ignore "target/debug/build/**" target/debug/ --output-path coveralls.json --output-type "coveralls+" --source-dir . --service-name travis-pro --service-job-id $TRAVIS_JOB_ID
+        - ./grcov --branch --ignore-not-existing --llvm --ignore "target/debug/build/** --ignore "/*" target/debug/ --output-path coveralls.json --output-type "coveralls+" --source-dir . --service-name travis-pro --service-job-id $TRAVIS_JOB_ID
```

Comment:
`--ignore "target/debug/build/** --ignore "/*"`
I think the `"` are wrong. Should probably be
`--ignore "target/debug/build/**" --ignore "/*"`

Reply:
🦅